lib/omniship/newgistics.rb

```diff
@@ -0,0 +1,29 @@
+require 'omniship/newgistics/track'
+
+module Omniship
+  module Newgistics
+    LABEL = "Newgistics"
+    TRACKING_REGEX = /TODO/
+    TRACKING_URL = "http://tracking.smartlabel.com/Default.aspx?TrackingValue="
+    BARCODE = 'Barcode'
+    REFERENCE_NUMBER = 'Reference Number'
```

Comment:
its the other option to pass into as qualifier below ```self.track(id, qualifier=BARCODE)``` we don't have any barcodes at the moment though...